### PR TITLE
Touch Ups

### PR DESCRIPTION
**Pull request by kimlisa**
resolves https://github.com/gravitational/webapps/issues/156

#### Description
- went through all loading screen views and fixed inconsistent loader placement
- merged missing commits from https://github.com/gravitational/webapps/pull/144 (displaying auth type on user table)
  - went through stratus merged PRs list with stratus git log to determine if there were other missing commits (none)
- fixed sorting bug:
  - ascending ^ now ascends (ie: abc), descending v now descends (cba)
- before registering user feature, checks if user has access
- session recording:
  - fixed search bar
  - increased page limit from 7 to 50

### CODE REVIEW

File: packages/teleport/src/AuthConnectors/AuthConnectors.tsx [hunk 1]
```diff
@@ -53,9 +53,14 @@ export default function AuthConnectors() {
...
-      <Flex justifyContent="center" alignItems="center" flex="1">
-        <Indicator />
-      </Flex>
+      <FeatureBox>
+        <FeatureHeader>
+          <FeatureHeaderTitle>Auth Connectors</FeatureHeaderTitle>
+        </FeatureHeader>
+        <Box textAlign="center" m={10}>
+          <Indicator />
+        </Box>
```
Comment (alex-kovoy) [inline packages/teleport/src/AuthConnectors/AuthConnectors.tsx:62]:
```ts
{isProcessing && (<Box textAlign="center" m={10}>
   <Indicator />
</Box>)
```

Put it under existing header, no need to duplicate it

File: packages/teleport/src/Roles/Roles.tsx [hunk 2]
```diff
@@ -54,9 +54,14 @@ export function Roles(props: ReturnType<typeof useRoles>) {
 
   if (isProcessing) {
     return (
-      <Flex justifyContent="center" alignItems="center" flex="1">
-        <Indicator />
-      </Flex>
+      <FeatureBox>
+        <FeatureHeader alignItems="center">
+          <FeatureHeaderTitle>Roles</FeatureHeaderTitle>
+        </FeatureHeader>
```
Comment (alex-kovoy) [inline packages/teleport/src/Roles/Roles.tsx:60]: the same here

File: packages/teleport/src/TrustedClusters/TrustedClusters.tsx [hunk 3]
```diff
@@ -53,9 +53,14 @@ export default function TrustedClusters() {
 
   if (tclusters.isProcessing) {
     return (
-      <Flex justifyContent="center" alignItems="center" flex="1">
-        <Indicator />
-      </Flex>
+      <FeatureBox>
+        <FeatureHeader alignItems="center">
+          <FeatureHeaderTitle>Trusted Clusters</FeatureHeaderTitle>
+        </FeatureHeader>
```
Comment (alex-kovoy) [inline packages/teleport/src/TrustedClusters/TrustedClusters.tsx:59]: here too

**Review by alex-kovoy — COMMENTED** [review]
Does it also include fixes to broken search and sorting listed on the ticket?

**Comment by kimlisa** [comment]
> Does it also include fixes to broken search and sorting listed on the ticket?

yes

Broken search: https://github.com/gravitational/webapps/pull/157/files#diff-12cf245676b3081000a4ca533b2038f1, the reason why it wasn't working was that we forgot to supply another dependency which was search value

Sorting: https://github.com/gravitational/webapps/pull/157/files#diff-37f67037b310d0caa795dd7c2245f63e
asc (init):
Screenshot from 2020-09-25 11-15-54

desc:
Screenshot from 2020-09-25 11-15-59

File: packages/teleport/src/AuthConnectors/AuthConnectors.tsx (hunk 1 above)
Comment (kimlisa) [reply packages/teleport/src/AuthConnectors/AuthConnectors.tsx:62]: revised

File: packages/teleport/src/Roles/Roles.tsx (hunk 2 above)
Comment (kimlisa) [reply packages/teleport/src/Roles/Roles.tsx:60]: revised

File: packages/teleport/src/TrustedClusters/TrustedClusters.tsx (hunk 3 above)
Comment (kimlisa) [reply packages/teleport/src/TrustedClusters/TrustedClusters.tsx:59]: revised

**Pull request merged**